pkg/cloud/vsphere/provisioner/govmomi/create.go

```diff
@@ -25,15 +25,15 @@ import (
 	"sigs.k8s.io/cluster-api/pkg/util"
 )
 
-func (pv *Provisioner) Create(cluster *clusterv1.Cluster, machine *clusterv1.Machine) error {
+func (pv *Provisioner) Create(ctx context.Context, cluster *clusterv1.Cluster, machine *clusterv1.Machine) error {
```

Comment:
@sidharthsurana just for my knowledge, i don't see this additional parameter: ctx not been used at this function, what is the purpose of it?

Reply:
this is defined in the main repo's actuator interface.  It must be here since we're implementing the interface.

Reply:
This is actually one of the new parameter that was added in the actuator interface. Right now in this PR I did not make use of it. However, later on we can actually start using this context in the logging, especially when we move to klog

Reply:
klog doesn't use context from what little I've seen of it.  I may open up a few PR to add some of the VIC logging features into klog.